extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -615,8 +615,10 @@ public function mailPassword() {
 			return;
 		}
 
-		if ( !empty( array_intersect( F::app()->wg->AccountAdminGroups, $user->getGroups() ) ) ) {
-			\Wikia\Logger\WikiaLogger::instance()->warning("Junior helper cannot change account info");
+		if ( !empty( array_intersect( F::app()->wg->AccountAdminGroups, $user->getAllGroups() ) ) ) {
```

Comment:
Why was this changed to `getAllGroups`?  The `getAllGroups` method returns *all* groups defined in the system, not just the groups for that user.